dGame/dGameMessages/GameMessages.cpp

```diff
@@ -5394,6 +5392,8 @@ void GameMessages::HandleRemoveItemFromInventory(RakNet::BitStream& inStream, En
 		const auto itemType = static_cast<eItemType>(item->GetInfo().itemType);
 		if (itemType == eItemType::MODEL || itemType == eItemType::LOOT_MODEL) {
 			item->DisassembleModel(iStackCount);
+		} else if (itemType == eItemType::VEHICLE) {
+			Database::Get()->DeleteUgcBuild(item->GetSubKey());
```

Comment:
Since this is "RemoveItem", what happens if you put a modular build in a vault? Is `bConfirmed` only for delete?

Reply:
i will check, good question

Reply:
given the line above this disassembles models into bricks, i dont think bConfirmed is used for vault transfers and this does destroy the item.  is this an apt conclusion?

Reply:
this sends RequestMoveItemBetweenInventoryTypes and not this message.  no concerns in this case.